iOS/Flows/Conversation/Cells/ConversationMessagesCell.swift

```diff
@@ -197,7 +197,7 @@ class ConversationMessagesCell: UICollectionViewCell, ConversationUIStateSettabl
             }.store(in: &self.subscriptions)
     }
 
-    func scrollToMessage(with messageId: MessageId, animateSelection: Bool) async {
+    func scrollToMessage(with messageId: MessageId, animateScroll: Bool, animateSelection: Bool) async {
```

Comment:
I wanted to speed navigation in the app and a lot of times it doesn't makes sense for a scroll animation, or an animated selection.